v2/go.mod

```diff
@@ -3,23 +3,23 @@ module github.com/rebuy-de/rebuy-go-sdk/v2
 go 1.13
 
 require (
-	github.com/aws/aws-sdk-go v1.28.9
+	github.com/aws/aws-sdk-go v1.32.0
 	github.com/gemnasium/logrus-graylog-hook/v3 v3.0.2
 	github.com/google/go-github/v30 v30.1.0
 	github.com/google/uuid v1.1.1
-	github.com/goreleaser/nfpm v1.1.10
+	github.com/goreleaser/nfpm v1.3.1
 	github.com/gorilla/securecookie v1.1.1
 	github.com/gorilla/sessions v1.2.0
 	github.com/julienschmidt/httprouter v1.3.0
-	github.com/mitchellh/mapstructure v1.1.2
+	github.com/mitchellh/mapstructure v1.3.2
 	github.com/pkg/errors v0.9.1
 	github.com/pmezard/go-difflib v1.0.0
-	github.com/sirupsen/logrus v1.4.2
-	github.com/spf13/cobra v0.0.5
-	github.com/tidwall/pretty v1.0.0
-	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
-	golang.org/x/sync v0.0.0-20190423024810-112230192c58
-	golang.org/x/tools v0.0.0-20200203023011-6f24f261dadb
-	gopkg.in/yaml.v2 v2.2.8
-	k8s.io/client-go v0.17.2
+	github.com/sirupsen/logrus v1.6.0
+	github.com/spf13/cobra v1.0.0
+	github.com/tidwall/pretty v1.0.1
+	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
+	golang.org/x/sync v0.0.0-20200317015054-43a5402ce75a
+	golang.org/x/tools v0.0.0-20200612022331-742c5eb664c2
+	gopkg.in/yaml.v2 v2.3.0
```

Comment:
Have you taken a look at the v3 package? Not sure what has changed
https://gopkg.in/yaml.v3

Reply:
Not sure if I want to do a major update when there is no changelog.

Reply:
Looking at the repo, we might want to replace it at some time.